src/pretix/presale/templates/pretixpresale/event/fragment_addon_choice.html

```diff
@@ -6,13 +6,14 @@
 {% load eventsignal %}
 {% load rich_text %}
 {% for c in form.categories %}
+    {% with category_idx=forloop.counter %}
     <fieldset data-addon-max-count="{{ c.max_count }}"{% if c.multi_allowed %} data-addon-multi-allowed{% endif %}>
         <legend>{{ c.category.name }}</legend>
         {% if c.category.description %}
             {{ c.category.description|rich_text }}
         {% endif %}
         {% if c.min_count == c.max_count %}
-            <p class="addon-count-desc">
+            <p class="addon-count-desc" id="addon-count-desc-{{ category_idx }}">
```

Comment:
This is a fragment template that will be included once per position, so these IDs won't really be unique, will they?

Reply:
🤦  You are right! I will add the `{{ form.pos.pk }}`. Thanks!